packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/FLTCam.m

```diff
@@ -251,6 +254,55 @@ - (instancetype)initWithMediaSettings:(FCPPlatformMediaSettings *)mediaSettings
   return self;
 }
 
+static void selectBestFormatForRequestedFrameRate(
```

Comment:
These helpers should be defined above their call sites; I'm suprised this is even compiling.